v4/source/channels.yaml

```diff
@@ -189,6 +189,42 @@
             // GetPublicChannelsByIdsForTeam
             channels, resp := Client.GetPublicChannelsByIdsForTeam(<TEAMID>, channelIds)
 
+  /channels/{channel_id}/timezones:
+    get:
+      tags:
+        - channels
+      summary: Get timezones in a channel
+      description: |
+        Get a list of timezones for the users who are in this channel.
```

Comment:
In addition to the other things I asked about before, this should note that it requires server 5.6 or higher

Reply:
done